README.md

```diff
@@ -1 +1,77 @@
-# Berry-IMU
\ No newline at end of file
+# Berry-IMU
+This module provides an implementation of the [BerryIMUv3](https://ozzmaker.com/product/berryimu-accelerometer-gyroscope-magnetometer-barometricaltitude-sensor/) using an i2c connection on ARM64 systems.
+
+## Setup
+
+###  Wiring
+Ensure i2c communication is enabled on your single board computer. Wire the SCL and SDA pins on the board to the corresponding pins on the IMU. 
+The IMU can be powered with the 3.3v pin. You can verify the i2c connection by running `i2cdetect -y 1` on your board. If the IMU is wired correctly, you will see 
+the i2c addresses `1c` and `6a`. 
+
+### Registry
+This module can be installed via the viam registry. 
+
+- Go to your robot's page on app.viam.com.
+- Click on the *Create Component* button in the Components section.
+- Search for the *berryimu* module and select it. 
+
+This will automatically install the module to your robot.
```

Comment:
Should I also have instructions on how to run locally?

Reply:
No it's okay - maybe point to the docs for running a module locally